tutorial/input_output.py

```diff
@@ -0,0 +1,199 @@
+                "False": "Correct! It can be used on any Python object.",
+                "True:": "Wrong! Try it in a cell below.",
+            },
+            correct_answer="False",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="What does the <code>f</code> before a string do? E.g <code>f'Hello {name}'</code>",
+            options={
+                "It formats the string as a float": "Wrong! Try it in a cell below.",
+                "It changes the color of the string": "Wrong! Try it in a cell below.",
+                "It allows inserting variables into a string": "Correct! In the example above, it will print the value of the <code>name</code> variable, if it is defined.",
+            },
+            correct_answer="It allows inserting variables into a string",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2])
+
+
+class StringInput(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="What does the function <code>input</code> do?",
+            options={
+                "It asks for user input and returns it as a string": "Correct! It is used to read user input from the console.",
+                "It shows a list of input devices available on the current computer.": "Wrong! It is used to read user input from the console.",
+            },
+            correct_answer="It asks for user input and returns it as a string",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="What happens if you call <code>input()</code> in the middle of a function?",
+            options={
+                "The function execution stops and it waits for the user to type an input": "Correct! Input is a blocking function which waits for user to enter a string in the console and press enter.",
+                "The function continues its execusion": "Wrong!",
+            },
+            correct_answer="The function execution stops and it waits for the user to type an input",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2])
+
+
+class Paths(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="What does the operator <code>/</code> do when applied to two <code>Pathlib.Path</code> objects?",
+            options={
+                "It removes the second path from the first": "Wrong, try it in the shell.",
+                "It concatenates paths": "Correct, it lets you construct a path from different segments",
+            },
+            correct_answer="It concatenates paths",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="If you use Pathlib,  do you need to use different path separators on Windows and Linux to combine path segments?",
+            options={
+                "No, you can combine Pathlib.Path objects with <code>/</code>": "Correct! Pathlib will then generate the correct path for your OS.",
```

Comment:
Unfortunately, in the replies, the `<code>` is not rendered.

```suggestion
                "No, you can combine Pathlib.Path objects with /": "Correct! Pathlib will then generate the correct path for your OS.",
```